ddtrace/tracer/option_test.go

```diff
@@ -129,4 +129,38 @@ func TestTracerOptionsDefaults(t *testing.T) {
 		assert.Equal("testEnv", c.globalTags[ext.Environment])
 		assert.True(c.debug)
 	})
+
+	t.Run("envs", func(t *testing.T) {
+		if err := os.Setenv("DD_SERVICE_NAME", "TEST_SERVICE"); err != nil {
+			panic("could not set environment variable DD_SERVICE_NAME during testing")
+		}
+		if err := os.Setenv("DD_TAGS", "env:test, aKey:aVal,bKey:bVal"); err != nil {
```

Comment:
Add a tag where there is only a key please, to cover for that switch case scenario.